ee/src/main/java/org/jboss/as/ee/subsystem/ManagedExecutorServiceRemove.java

```diff
@@ -36,8 +34,4 @@ private ManagedExecutorServiceRemove() {
         super(ManagedExecutorServiceAdd.INSTANCE);
     }
 
-    @Override
-    protected ServiceName serviceName(String name) {
```

Comment:
Given that you are no longer overriding behavior of ServiceRemoveStepHandler, we can eliminate this class altogether.

Reply:
done